python/foglamp/plugins/storage/postgres/backup_restore/lib.py

```diff
@@ -7,11 +7,16 @@
 import subprocess
 import time
 import os
+import asyncio
+import json
 from enum import IntEnum
+import psycopg2
```

Comment:
Can I suggest we have a comment somewhere explain why this script uses the database directly and does not go via storage

Reply:
@MarkRiddoch has a very valid point.
I know our testing and build environment is not ready, but we must be 100% sure whatever we develop it works in:
1. development env (i.e. after make)
2. deployment env (i.e. after sudo make install in /usr/local/foglamp)
3. snap on Ubuntu x86_64
4. snap on Ubuntu Core armhf

I know it sounds long and complicated, but it is not - really. Let's talk Monday about that.
/iz

Reply:
@MarkRiddoch, done